### PR TITLE
🌐 Add Japanese translation for Advanced User Guide - Additional Status Codes

### PR DESCRIPTION
**Pull request by Attsun1031**
This PR translates the advanced/additional-status-codes.md to Japanese.
Please review advanced/additional-status-codes.md and ja/mkdocs.yml.

### CODE REVIEW

File: docs/ja/docs/advanced/additional-status-codes.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+それは、デフォルトのステータスコードか、 *path operation* でセットしたものを利用します。
+
+## 追加のステータスコード
+
+メインのステータスコードとは別に追加のステータスコードを返したい場合は、追加のステータスコードが設定された `JSONResponse` のような `Response` を直接返します。
+
+例えば、itemを更新し、成功した場合は200 "OK"のHTTPステータスコードを返す *path operation* を作りたいとします。
+
+しかし、新しいitemも許可したいとします。itemが存在しない場合は、それらを作成して201 "Created"を返すとします。
```
Comment (komtaki) [inline docs/ja/docs/advanced/additional-status-codes.md:13]:
```suggestion
しかし、新しいitemも許可したいです。itemが存在しない場合は、それらを作成して201 "Created"を返します。
```

You can see from the previous sentence that it is a hypothetical story. So I think this is more natural.

File: docs/ja/docs/advanced/additional-status-codes.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+{!../../../docs_src/additional_status_codes/tutorial001.py!}
+```
+
+!!! warning "注意"
+    上記の例のように `Response` を直接返す場合、それは直接返されます。
+
+    モデルなどはシリアライズされません。
+
+    必要なデータが含まれていることや、値が有効なJSONであること (`JSONResponse` を使う場合) を確認してください。
+
```
Comment (komtaki) [inline docs/ja/docs/advanced/additional-status-codes.md:27]:
```suggestion
```

The original have the one new line.

**Review by komtaki — COMMENTED** [review]
Thank you for the translate.
I commented a few minor suggestion. 👍

**Comment by Attsun1031** [comment]
@komtaki Thank you for your review! I fixed everything, so please check again.

**Review by komtaki — APPROVED** [review]
LGTM 👌

**Comment by Attsun1031** [comment]
I resolved all conflicts. (All were mkdocs nav order)
https://github.com/tiangolo/fastapi/pull/2145/commits/6214f7492e15693d66628eb15f94c0152abc2798

File: docs/ja/docs/advanced/additional-status-codes.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# 追加のステータスコード
+
+デフォルトでは、 **FastAPI** は `JSONResponse` を使ってレスポンスを返します。その `JSONResponse` の中には、 *path operation* が返した内容が入ります。
+
+それは、デフォルトのステータスコードか、 *path operation* でセットしたものを利用します。
+
+## 追加のステータスコード
+
+メインのステータスコードとは別に追加のステータスコードを返したい場合は、追加のステータスコードが設定された `JSONResponse` のような `Response` を直接返します。
```
Comment (tokusumi) [inline docs/ja/docs/advanced/additional-status-codes.md:9]:
```suggestion
メインのステータスコードとは別に、他のステータスコードを返したい場合は、`Response` (`JSONResponse` など) に追加のステータスコードを設定して直接返します。
```

File: docs/ja/docs/advanced/additional-status-codes.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+しかし、新しいitemも許可したいです。itemが存在しない場合は、それらを作成して201 "Created"を返します。
+
+これを達成するには、 `JSONResponse` をインポートし、 `status_code` を設定して直接内容を返します。
+
+```Python hl_lines="4  23"
+{!../../../docs_src/additional_status_codes/tutorial001.py!}
+```
+
+!!! warning "注意"
+    上記の例のように `Response` を直接返す場合、それは直接返されます。
```
Comment (tokusumi) [inline docs/ja/docs/advanced/additional-status-codes.md:22]:
```suggestion
    上記の例のように `Response` を明示的に返す場合、それは直接返されます。
```
自然に

File: docs/ja/docs/advanced/additional-status-codes.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    必要なデータが含まれていることや、値が有効なJSONであること (`JSONResponse` を使う場合) を確認してください。
+
+!!! note "技術詳細"
+    `from starlette.responses import JSONResponse` を利用することもできます。
+
+    **FastAPI** は `fastapi.responses` と同じ `starlette.responses` を、開発者の利便性のために提供しています。しかし有効なレスポンスはほとんどStarletteから来ています。 `status` についても同じです。
+
+## OpenAPIとAPIドキュメント
+
+もし追加のステータスコードとレスポンスを直接返す場合、それらはOpenAPIスキーマ (APIドキュメント) には含まれません。なぜなら、FastAPIは何が返されるのか事前に知ることができないからです。
```
Comment (tokusumi) [inline docs/ja/docs/advanced/additional-status-codes.md:35]:
```suggestion
ステータスコードとレスポンスを直接返す場合、それらはOpenAPIスキーマ (APIドキュメント) には含まれません。なぜなら、FastAPIは何が返されるのか事前に知ることができないからです。
```
意味が通るなら、できる限り、additionalは訳さない方が読みやすいと思います。

**Review by tokusumi — CHANGES_REQUESTED** [review]
Thank you for your work! I'd like to propose some feedbacks.

**Review by tokusumi — APPROVED** [review]

**Comment by tiangolo** [comment]
Thanks a lot @Attsun1031 for your work! :cake: :rocket: 

And thanks @tokusumi and @komtaki for the reviews! :bow: :coffee:

**Pull request merged**